### ISSUE TITLE
Confused about dependent/independent experiment

### ISSUE THREAD

**Issue by Dray-Xu** [RespectKnowledge/EEG_Speech_Depression_MultiDL#1]
Hi, dear contributor. 
    Respect to your sharing! 

    But I got some confusion that -- If some fragments of an individual already appear in the test set, will the rest of his fragments appear in the training set possibly? Or is this experiment dependent/independent?

Thanks for your attention!

**Comment by RespectKnowledge** [comment]
Thanks for your interest. Please keep in mind. The dataset for training and testing select first and then segments. Otherwise there is no make sense. So sorry for late response.

**Comment by Dray-Xu** [comment]
I mean, in the same round of training and testing, do clips from the same subject only appear in one of the training or testing sets? To avoid information leakage.

Thanks for your reply!!!  😄

**Comment by RespectKnowledge** [comment]
Yes, thanks

**Comment by RespectKnowledge** [comment]
complete the issue
